collector.yml

```diff
@@ -0,0 +1,4 @@
+DatabaseDSN: postgres://collector:ZnPfJ!4vTb-nURv8M!JkkuAcjYLYbkiX@localhost:5433
```

Comment:
should these passwords be in here and in sqlboiler.toml?

Reply:
or are these just for testing/docker?

Reply:
These are for local testing. Production will be completely different